### PR TITLE
DevKit updates for master branch

### PR DESCRIPTION
**Pull request by ElectricMaxxx**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,10 +26,9 @@ list:
 	@echo 'unit_tests:               will run unit tests only'
 	@echo 'functional_tests_phpcr:  will run functional tests with PHPCR'
 
-	@echo 'test_installation:    will run installation test'
+
 include ${TESTING_SCRIPTS_DIR}/make/unit_tests.mk
 include ${TESTING_SCRIPTS_DIR}/make/functional_tests_phpcr.mk
-include ${TESTING_SCRIPTS_DIR}/make/test_installation.mk
```
Comment (dbu) [inline Makefile]: are you giving up on the test installation step?

Comment (dbu) [inline Makefile]: did you give up on the test installation?

Comment (dbu) [inline Makefile]: did you give up on the installation test?

**Comment by dbu** [comment]
did you give up on the installation test?

**Comment by ElectricMaxxx** [comment]
@dbu yes, atm it gives us less value and can confuse. At the end i would like to have one integration test (into an app equal to sandbox) of each package on one matrix item. Doing it that way we will see earlier, when something fails.

**Comment by dbu** [comment]
okay, we can do it that way. 

i found it appealing that we would see problems right away on the specific repository when a pull request e.g. accidentally introduces a dependency on another library instead of either adding it to the composer requirements (and not dev-require) or handling the absence of the library gracefully.

but its your call, if the separate setup is easier to do, feel free to drop this from here. we never really used that check yet, so we are not losing anything.

File: Makefile (hunk 1 above)
Comment (ElectricMaxxx) [reply Makefile]: :+1:

**Pull request merged**